api/classes/base_model.js

```diff
@@ -1,13 +1,19 @@
-var Bookshelf = require('./database').Bookshelf;
+"use strict";
 
-var instanceProps = {
-  column: function(columnName, alias, escapeString) {
-    var prefix = escapeString ? '"' : '';
-    var column = prefix + this.tableName + prefix + '.' + prefix + columnName + prefix;
-    return alias ? column + ' AS ' + prefix + alias + prefix : column;
+const Bookshelf = require(`./database`).Bookshelf;
+
+const instanceProps = {
+  column(columnName, alias, escapeString) {
+    const prefix = escapeString ? `"` : ``;
+    const column = `${prefix}${this.tableName}${prefix}` +
+                   `.` +
```

Comment:
I'd argue not to split, but that's because I use widescreen text editors, and if a line is code, in which which the placement matters, I don't mess with that placement. If a string really is *that* long, that's usually a sign of trying to do too much in the wrong place. For instance:
```
let tableName = `${prefix}${this.tableName}${prefix}`;
let fieldName = `${prefix}${columnName}${prefix}`;
const column = `${tableName}.${fieldName}`;
```
boom: clean, short, sorted.